.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.1.2
+current_version = 0.1.3
```

Comment:
I re-enabled the auto semantic versioning in travis.yml, so you shouldnt need to update this as it'll be done automatically after your PR is merged. But note, you've used `feat(toolchain) ...` in the commit message, which will cause a bump to version `0.2.2` in the auto sem-ver build. If you remove all the version bump stuff and use `git commit --amend --signoff`, then change the commit message to `fix(toolchain) ...`, the resulting new version will be `0.1.3`